html/objects/_flag.scss

```diff
@@ -132,15 +115,15 @@
 }
 
 @media all and (max-width: $sprk-flag-stacked-breakpoint) {
-  
+
   /// Stacked flag at smaller screen width.
   .sprk-o-Flag--stacked {
-    direction: ltr;
+    flex-flow: column;
```

Comment:
If we are only setting column, we dont need the whole shorthand `flex-flow`. Can just just `flex-direction`

Reply:
You're right!